checker/raw_result.go

```diff
@@ -61,6 +61,19 @@ type DependencyUpdateToolData struct {
 	Tools []Tool
 }
 
+// WebhooksData contains the raw results
+// for the Webhook check.
+type WebhooksData struct {
+	Webhook []WebhookData
+}
+
+// WebhookData contains the raw results
+// for webhook check.
+type WebhookData struct {
+	ID        *int64
+	HasSecret *bool
```

Comment:
Nit: should we maybe name this to something like `UsesAuthSecret` or `UsesAuthToken`? It'll be bit more generic instead of being GitHub-specific and will make it the code more readable.